cmd/testServer/main.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"google.golang.org/grpc"
+)
+
+const BUCKETNAME = "secrets"
```

Comment:
Please avoid secrets in the code database

Reply:
it's not a secret. 
in addition, testServer/ is a development and debugging package. does not carry a functional load for services